src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -683,7 +683,7 @@ static void addERR_ParamsCantBeWithModifier(BindingDiagnosticBag diagnostics, Sy
                 // error CS0721: '{0}': static types cannot be used as parameters
                 diagnostics.Add(
                     ErrorFacts.GetStaticClassParameterCode(parameter.ContainingSymbol.ContainingType?.IsInterfaceType() ?? false),
```

Comment:
>GetStaticClassParameterCode [](http://example.com/codeflow?start=31&length=27)

It looks like there is a code in src\Compilers\CSharp\Portable\Symbols\Source\SourcePropertyAccessorSymbol.cs that reports the same error and can benefit from similar adjustment.

Reply:
Yes, I know. For properties, I thought that it makes sense to report the error on the setter, because it's obvious which type is the culprit (the type of the property), but it might not be obvious whether `get` or `set` method is the problem.

The easiest way to implement what you suggest is probably by making `SourcePropertySymbolBase.TypeLocation` `internal`: 4537a4b59ba8119e6aea2f7b34add5f124786aa8